airbyte-integrations/connectors/source-mysql/src/testFixtures/java/io/airbyte/integrations/source/mysql/MySQLTestDatabase.java

```diff
@@ -70,13 +79,34 @@ public MySQLTestDatabase withoutStrictMode() {
   }
 
   static private final int MAX_CONNECTIONS = 1000;
+  private String databaseName = "";
+
+  @Override
+  public String getDatabaseName() {
+    if (databaseName.isBlank()) {
+      return super.getDatabaseName();
+    } else {
+      return databaseName;
+    }
+  }
+
+  @Override
+  public void close() {
+    super.close();
+    databaseName = "";
+  }
+
+  public void setDatabaseName(final String databaseName) {
+    this.databaseName = databaseName;
+  }
 
   @Override
   protected Stream<Stream<String>> inContainerBootstrapCmd() {
+    System.out.println("name: " + String.format("CREATE DATABASE `%s`", getDatabaseName()));
     return Stream.of(mysqlCmd(Stream.of(
         String.format("SET GLOBAL max_connections=%d", MAX_CONNECTIONS),
-        String.format("CREATE DATABASE %s", getDatabaseName()),
-        String.format("CREATE USER '%s' IDENTIFIED BY '%s'", getUserName(), getPassword()),
+        String.format("CREATE DATABASE \\`%s\\`", getDatabaseName()),
```

Comment:
Why this change in a test class? Why the backticks? Why the change bellow to the `CREATE USER`
Are we adding a test case for this?